default-gecko-config

```diff
@@ -15,8 +15,34 @@ else
 ac_add_options --target=arm-linux-androideabi
 fi
 
+# Follow the section 'Bootstrap dependencies' at
+#
+#  https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Build_Instructions/Simple_Firefox_for_Android_build
+#
+# for installing the Android SDK and NDK.
+
+ac_add_options --with-android-sdk="$HOME/.mozbuild/android-sdk-linux"
+ac_add_options --with-android-ndk="$HOME/.mozbuild/android-ndk-r11b"
```

Comment:
Yes NDK and SDK are well installed, I probably missed something because I only applied that patch.